src/pkg/token/token.go

```diff
@@ -34,8 +35,8 @@ type Token struct {
 
 // New ...
 func New(opt *Options, claims jwt.Claims) (*Token, error) {
-	err := claims.Valid()
-	if err != nil {
+	var v = jwt.NewValidator(jwt.WithLeeway(60 * time.Second))
```

Comment:
Since we align with upstream distribution/distribution token convention, would it better to use it leeway as `jwt.WithLeeway(token.Leeway)`  in case of we need track the upstream changes manually in the future.

Reply:
Fixed by a788b68